rinja/src/lib.rs

```diff
@@ -23,41 +23,6 @@
 //! The template syntax is very similar to [Jinja](http://jinja.pocoo.org/),
 //! as well as Jinja-derivatives like [Twig](http://twig.sensiolabs.org/) or
 //! [Tera](https://github.com/Keats/tera).
```

Comment:
Would be nice to mention that the main element is the `template` attribute and to link to it.